src/lib/gundb/UserPropertiesClass.js

```diff
@@ -13,14 +13,16 @@ const logger = pino.child({ from: 'UserStorage' })
 export default class UserProperties {
   /**
    * Default User Properties
-   * @type {{isMadeBackup: boolean, addBackupFeed: boolean}}
+   * @type {{isMadeBackup: boolean, addBackupFeed: boolean, firstVisitApp:number, EToro_needAddCardSpending:boolean}}
```

Comment:
refactor to shorter name
start with lowercase, dont use _

Reply:
I rename to etoroNeedAddCardSpending